lib/migration-chunks/slicers/into-separate-chunk.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const createPredicateSlicer = require('./predicate');
+
+const returnTrueOnce = function () {
+  let neverCalled = true;
+  return function () {
+    const ret = neverCalled;
+    neverCalled = false;
+    return ret;
+  };
+};
+
+module.exports = (itemType) => {
+  const isItemType = (item) => item.type === itemType;
+  return createPredicateSlicer((list) => {
+    if (isItemType(list[0])) {
+      return returnTrueOnce();
+    }
```

Comment:
nit and a bit OT to this PR but could we fix the eslint rules to require an empty line after `if` blocks?

Reply:
I'm not sure this is something that needs to be regulated by a linter rule but we can discuss it in the team I guess?

Reply:
we can discuss it in the team and if we agree it's something we want we should put in on the linter. Otherwise we'll have this conversation next time someone forgets to add the whiteline : )